content/tracing/setup/dotnet.md

```diff
@@ -14,9 +11,15 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-Tracing for .NET applications is coming soon from Datadog. The officially supported beta will launch as early as August 2018. To be notified of the beta launch, <a href="https://goo.gl/forms/SCKOOlHS7tNzyMt93">submit this form</a> <br>.
+The APM tracer for .NET applications is currently in alpha and not officially supported by Datadog. The officially supported beta will launch as early as August 2018. We do not recommend running the alpha tracer in production.
```

Comment:
the alpha actually isn't available yet, so can we change this language back to: 

Tracing for .NET applications is coming soon from Datadog. The officially supported beta will launch as early as August 2018. To be notified of the beta launch, fill out the survey below: